doc/source/parameters/cfd/multiphysics.rst

```diff
@@ -5,19 +5,20 @@ This subsection defines the multiphysics interface of Lethe and enables the solu
 .. code-block:: text
 
   subsection multiphysics
-    set fluid dynamics 		= true
+    set fluid dynamics = true
 
     # Thermal physics
-    set heat transfer 		= false
-    set viscous dissipation 	= false
-    set buoyancy force 		= false
-    set use average velocity field = false
+    set heat transfer       = false
+    set viscous dissipation = false
+    set buoyancy force      = false
 
     # Tracer
-    set tracer 			= false
+    set tracer = false
 
     # Multiphase flow
-    set VOF 			= false
+    set VOF = false
+
+    set use time average velocity field = false
```

Comment:
why change the alignment here?

Reply:
I ran this subsection in prmindent and copied the output. I can put it back the way it was though, the alignment was fine.